commons/monitor/questionnaire/questionnaire_application_usage_log.avsc

```diff
@@ -0,0 +1,19 @@
+{
+  "namespace": "org.radarcns.monitor.questionnaire",
+  "type": "record",
+  "name": "QuestionnaireApplicationUsageLog",
+  "doc": "Schema of reporting events when opening the app, clicking notifications, and starting and finish questionnaires.",
+  "fields": [
+    { "name": "time", "type": "double", "doc": "Device timestamp in UTC (s)." },
+    { "name": "timeReceived", "type": "double", "doc": "Device receiver timestamp in UTC (s)." },
+    { "name": "packageName", "type": "string", "doc": "Package name of the launched app." },
+    { "name": "categoryName", "type": ["null", "string"], "doc": "App category as given by the play store, null if a category is not listed or unable to be fetched.", "default": null },
+    { "name": "categoryNameFetchTime", "type": ["null", "double"], "doc": "Timestamp in UTC when the category was attempted to fetch from the play store (s), null if not fetched.", "default": null },
+    { "name": "eventType", "type": {
+      "name": "UsageEventType",
+      "type": "enum",
+      "doc": "Android activity usage event type.",
+      "symbols": ["APP_OPEN_NOTIFICATION", "APP_OPEN_DIRECTLY", "APP_FOREGROUND_NOTIFICATION", "QUESTIONNAIRE_STARTED", "QUESTIONNAIRE_COMPLETED", "OTHER", "UNKNOWN"]
```

Comment:
Can we also add a symbol for questionnaire closed ? For if someone closes the questionnaire in between filling it?